manifests/server.pp

```diff
@@ -71,6 +71,12 @@
   Boolean $ssl_weak_cert                                        = false,
   Boolean $ssl_invalid_hostnames                                = false,
   Enum['requireSSL', 'preferSSL', 'allowSSL'] $ssl_mode         = 'requireSSL',
+  Optional[Boolean] $tls                                        = undef,
```

Comment:
Can this be?
```suggestion
  Boolean $tls                                                  = false,
```

Reply:
This should not break anything.
I'm not able to test it with a real environment.